sirepo_bluesky/sirepo_bluesky.py

```diff
@@ -70,8 +75,9 @@ def auth(self, sim_type, sim_id):
 
     def copy_sim(self, sim_name):
         """ Create a copy of the current simulation. Returns a new instance of SirepoBluesky. """
-        assert self.sim_id
-        # simulationId, simulationType, name, folder
+        if not self.sim_id:
+            raise ValueError(f'sim_id is {repr(self.sim_id)}')
```

Comment:
I don't think we need `repr(...)` here, the f-string will take care of the correct representation.
```suggestion
            raise ValueError(f'sim_id is {self.sim_id}')
```

Reply:
Addressed via https://github.com/NSLS-II/sirepo-bluesky/pull/89/commits/392a5b47c19cc891dc37b06d46c310832a3aa340.

Reply:
Actually, I see why you did it, and I reverted it in a slightly different way - with `f{...!r}`. The original idea was clever.

Here is a comparison:

```py
In [11]: val = None

In [12]: print(f"val: {val!r}")
val: None

In [13]: val = False

In [14]: print(f"val: {val!r}")
val: False

In [15]: val = None

In [16]: print(f"val: {repr(val)}")
val: None

In [17]: val = False

In [18]: print(f"val: {repr(val)}")
val: False

In [19]: val = ''

In [20]: print(f"val: {repr(val)}")
val: ''
```